ProcessMaker/Events/TokenCreated.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ProcessMaker\Events;
+
+use Illuminate\Foundation\Events\Dispatchable;
+use Laravel\Passport\Token;
+use ProcessMaker\Contracts\SecurityLogEventInterface;
+
+class TokenCreated implements SecurityLogEventInterface
+{
+    use Dispatchable;
+
+    public $data;
+    public $changes;
+
+    /**
+     * Create a new event instance.
+     *
+     * @return void
+     */
+    public function __construct(Token $created_values)
```

Comment:
First, update the branch with the latest feature/FOUR-8121 changes.

Considering the acceptance criteria, would you consider making the following changes?

```php
<?php

namespace ProcessMaker\Events;

use Illuminate\Broadcasting\InteractsWithSockets;
use Illuminate\Foundation\Events\Dispatchable;
use Illuminate\Queue\SerializesModels;
use Illuminate\Support\Facades\Auth;
use Laravel\Passport\Token;
use ProcessMaker\Contracts\SecurityLogEventInterface;

class TokenCreated implements SecurityLogEventInterface
{
    use Dispatchable, InteractsWithSockets, SerializesModels;

    public $user;
    public $data;
    public $changes;

    /**
     * Create a new event instance.
     *
     * @param Token $token
     */
    public function __construct(Token $token)
    {
        $this->user = Auth::user();
        $this->data = [
            "Token Id" => $token->getKey()
        ];
        $this->changes = $token->toArray();
    }

    public function getChanges(): array
    {
        return $this->changes;
    }

    public function getData(): array
    {
        return $this->data;
    }

    public function getEventName(): string
    {
        return 'TokenCreated';
    }
}

```